pkg/server/container_create.go

```diff
@@ -416,12 +416,14 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 
```

Comment:
Add unit test cases for this logic.

Reply:
Can we cover this in an integrated test (in a follow-up PR)?